Python/Searching/BinarySearch.py

```diff
@@ -0,0 +1,14 @@
+def BinarySearch(List,key,start,end):

```

Comment:
Please review once again. You are not doing recursion and also the condition to check whether left>right pointer